features/kiesrecht.feature

```diff
@@ -7,116 +7,79 @@ Functionaliteit: Kiesrecht
+		| einddatum uitsluiting onbekend     | 1     | false              | 00000000                              | false             | OnbekendDatum  |            |                |                 |
+		| persoon ontvangt oproep            | 2     | true               |                                       | true              |                |            |                |                 |
+
+Rule: Uitsluiting van kiesrecht wordt alleen opgenomen wanneer de einddatum uitsluiting in de toekomst ligt
+	- Wanneer alleen het jaar van de einddatum uitsluiting bekend is, dan wordt de uitsluiting opgenomen tot en met dat jaar.
+	- Wanneer het jaar en de maand van de einddatum uitsluiting bekend is, en de dag niet, dan wordt de uitsluiting opgenomen tot en met die maand.
+	- Een volledig onbekende datum wordt hetzelfde geïnterpreteerd en weergegeven als het niet aanwezig zijn van die datum.
+
+	Abstract Scenario: kiesrecht <omschrijving>
+		Gegeven het systeem heeft een persoon met de volgende gegevens
+		| naam                        | waarde     |
+		| burgerservicenummer         | 999990001  |
+		En de persoon heeft de volgende kiesrecht gegevens
+		| naam                                  | waarde                  |
+		| uitgeslotenVanKiesrecht (38.10)       | <uitsluiting kiesrecht> |
+		| einddatumUitsluitingKiesrecht (38.20) | <einddatum uitsluiting> |
+		Als de persoon op 15 maart 2022 wordt geraadpleegd met de volgende parameters
+		| naam                | waarde                          |
+		| type                | RaadpleegMetBurgerservicenummer |
+		| burgerservicenummer | 999990001                       |
+		| fields              | kiesrecht                       |
+		Dan bevat de persoon met burgerservicenummer '999990001' de volgende 'kiesrecht' gegevens
+		| naam                                | waarde                                        |
+		| uitgeslotenVanKiesrecht             | <uitgeslotenVanKiesrecht>                     |
+		| einddatumUitsluitingKiesrecht.type  | <type>                                        |
+		| einddatumUitsluitingKiesrecht.datum | <einddatum>                                   |
+		| einddatumUitsluitingKiesrecht.jaar  | <einddatum jaar>                              |
+		| einddatumUitsluitingKiesrecht.maand | <einddatum maand>                             |
+
+		Voorbeelden:
+		| omschrijving                       | 38.10 | uitsluiting kiesrecht | einddatum uitsluiting | uitgeslotenVanKiesrecht | type           | einddatum  | einddatum jaar | einddatum maand |
+		| niet opgegeven                     |       |                       |                       |                         |                |            |                |                 |
+		| uitgesloten zonder einddatum       | A     | true                  |                       | false                   |                |            |                |                 |
```

Comment:
Moet uitgeslotenVanKiesrecht hier waar het in deze tabel nu op false staat niet true zijn ?

Reply:
nee, de persoon is uitgesloten van kiesrecht, dus uitgeslotenVanKiesrecht is waar (true)

Reply:
gaat over 5e kolom "uitgeslotenVanKiesrecht". waar false staat moet true komen

Reply:
gaat over 5e kolom "uitgeslotenVanKiesrecht". waar false staat moet true komen